setup.py

```diff
@@ -15,9 +15,10 @@
 INSTALL_REQUIRES = [
     "loguru==0.5.3",
     "requests==2.25.1",
-    "tqdm==4.56.0",
+    "tqdm==4.49",
```

Comment:
i've temporarily lowered the version because it conflicts with the version `<4.5` required by `datasets`.

Reply:
seems like we have a problem bumping `tqdm` on `datasets` because it introduces a windows permission error for v0.5.0+ 😬 : https://app.circleci.com/pipelines/github/huggingface/datasets/235/workflows/cfb6a39f-68eb-4802-8b17-2cd5e8ea7369/jobs/1111

Reply:
im fine by that